site/src/pages/HealthPage/WorkspaceProxyPage.tsx

```diff
@@ -138,14 +142,17 @@ export const WorkspaceProxyPage = () => {
                   color: theme.palette.text.secondary,
                 }}
               >
-                {warnings.length > 0 ? (
+                {warnings.length === 0 && errors.length === 0 ? (
+                  <span>OK</span>
+                ) : (
                   <div css={{ display: "flex", flexDirection: "column" }}>
+                    {errors.map((error, i) => (
+                      <span key={i}>{error}</span>
+                    ))}
                     {warnings.map((warning, i) => (
                       <span key={i}>{warning}</span>
                     ))}
```

Comment:
I tried turning this into an `<ul>` and playing around with `list-style-type` to prefix these with 🚫 or ⚠️ respectively, but this didn't seem to do what I wanted.